### PR TITLE
adds a new tip informing people how to use altar of god

### PR DESCRIPTION
**Pull request by boodaliboo**
good way to inform new players

Changelog
🆑
rscadd: Added a new tip
/🆑

### CODE REVIEW

**Comment by patpol4** [comment]
Bood you literally wrote one sentence how do you not spellcheck it

File: strings/tips.txt [hunk 1]
```diff
@@ -258,3 +258,4 @@ Just because someone attacks you doesn't mean they're trying to kill you, people
 Questions? Consult the Wiki: https://wiki.yogstation.net/wiki/Main_Page
 Questions? Consult the Mentors using the 'MHelp' system.
 Starving makes you move slower. Be sure to keep fed!
+As the Chaplain You can use your bible on the altar of god to choose a veriety of religions with special rites.
```
Comment (patpol4) [inline strings/tips.txt]: As the Chaplain, You can use your bible on the altar of god to choose a variety of religions with special rites.

**Comment by Marmio64** [comment]
i do enjoy a veriety of religions, good sir

File: strings/tips.txt [hunk 2]
```diff
@@ -258,3 +258,4 @@ Just because someone attacks you doesn't mean they're trying to kill you, people
 Questions? Consult the Wiki: https://wiki.yogstation.net/wiki/Main_Page
 Questions? Consult the Mentors using the 'MHelp' system.
 Starving makes you move slower. Be sure to keep fed!
+As the Chaplain, You can use your bible on the altar of god to choose a variety of religions with special rites.
```
Comment (JamieD1) [inline strings/tips.txt]:
```suggestion
As the Chaplain, You can use your bible on the "Altar of God" to choose a variety of religions with special rites.
```

**Pull request merged**